homeassistant/components/hassio/services.yaml

```diff
@@ -111,3 +111,11 @@ snapshot_partial:
       example: "password"
       selector:
         text:
+
+restore_full:
+  name: Restore from full backup.
+  description: Restore from full backup.
+
+restore_partial:
+  name: Restore from partial backup.
+  description: Restore from partial backup.
```

Comment:
Likewise the additional fields for a partial restore.

https://github.com/home-assistant/core/blob/cf9cc99984937f39a0ee4c5904919f0e7c86dea6/homeassistant/components/hassio/__init__.py#L119-L125